docs/ftue/01_prompt_an_image/FTUE_01_prompt_an_image.md

```diff
@@ -1,18 +1,24 @@
 # Prompt an Image
 
-Welcome to the second tutorial in our Griptape Nodes series! In this guide, you'll learn how to generate images using AI through the powerful GenerateImage node.
+Welcome to the second tutorial in our Griptape Nodes series! This guide focuses on getting familiar with the GenerateImage node.
```

Comment:
When we call out specific nodes, I think it's useful to include links to the node reference in case people are interested in learning more about the node.

You can create a new task for this & come back to it later if you like.